CHANGES.md

```diff
@@ -19,6 +19,7 @@ Fixed Issues:
 * [#842](https://github.com/ckeditor/ckeditor-dev/issues/842): Fixed: List style not restored when toggling list indent level in [Indent List](http://ckeditor.com/addon/indentlist) plugin.
 * [#711](https://github.com/ckeditor/ckeditor-dev/issues/711): Fixed: Dragging widgets should only work with left mouse button.
 * [#862](https://github.com/ckeditor/ckeditor-dev/issues/862): Fixed: "Object Styles" group in [Styles Combo](https://ckeditor.com/addon/stylescombo) plugin is visible only if whole element is selected.
+* [994](https://github.com/ckeditor/ckeditor-dev/pull/994): Fixed: Typo in CKEDITOR.focusManager API documentation. Thanks to [benjy](https://github.com/benjy)!
```

Comment:
Missing `#` char before the number.

Reply:
`CKEDITOR.focusManager` should be wrapped with backticks and linked to the API docs.